databases/migrations/chat-plugins/v5.sql

```diff
@@ -0,0 +1,20 @@
+-- Creates stats tables for the Perspective moderation tool.
+BEGIN TRANSACTION;
+
+CREATE TABLE perspective_stats (
+	staff TEXT NOT NULL,
+	roomid TEXT NOT NULL PRIMARY KEY,
+	result TINYINT(1) NOT NULL,
+	date TEXT NOT NULL, -- YYYY-MM-DD - for easy stat searching.
+	timestamp INTEGER NOT NULL
```

Comment:
Why do we store both the date and the timestamp? 

I'm not sure about the details of SQLite's performance, but I would imagine that an integer bounds comparison is probably faster than string pattern matching.

Reply:
Yeah, I think it was because I thought it was easier to just search for a date. Probably best to make it just do a stupidly long query to compare integers.

Reply:
Updated.